frontend/src/libs/components/button/button.tsx

```diff
@@ -10,12 +10,13 @@ type Properties = {
   icon?: IconType;
   size?: 'big' | 'small';
   style?: 'primary' | 'secondary' | 'plain';
-  type?: 'button' | 'submit';
+  type?: 'button' | 'submit' | undefined;
```

Comment:
`undefined` shouldn't be a value that could be passed as `type` prop value
```suggestion
  type?: 'button' | 'submit';
```

Reply:
![image](https://user-images.githubusercontent.com/106602282/231441478-2ce5bb2b-f74c-4f62-b24f-d98a0e924065.png)
In cases without "undefind" I have an error that I do not understand how to fix

Reply:
![image](https://user-images.githubusercontent.com/106602282/231540552-718f2de5-4918-42cf-a448-482090c1dea9.png)
Without undefine, a typing error appears that cannot be fixed. If you remove the question mark from the type, then you need to forward the type to 30 files